Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ModelData.xml

```diff
@@ -8897,6 +8897,18 @@
         <TechPurchaseCamera value="Star2CameraLow15"/>
         <UnitGlossaryCamera value="Star2CameraLow15"/>
     </CModel>
+    <CModel id="AP_ImmortalAttackPurifierImpact" parent="ImpactFX">
+        <Model value="Assets\Effects\Protoss\Immortal_Collection_Missile_Impact\Immortal_Collection_Missile_Impact.m3"/>
+        <EditorCategories value="Race:Protoss"/>
+    </CModel>
+    <CModel id="AP_PerfectedPowerNova" parent="OneShotSpellFX">
+        <Model value="Assets\Effects\Protoss\Adept_Collection_Purifier_RicochetGlaive_Missile_Impact\Adept_Collection_Purifier_RicochetGlaive_Missile_Impact.m3"/>

```

Comment:
This references the value in the gamefiles (no `AP/` prefix) and thus doesn't have to be updated. The file should not be added, though.

Reply:
Thank you for catching that, I thought I removed that file in my recent pull requests but appears I forgot to.